features/attr.yml

```diff
@@ -0,0 +1,19 @@
+name: attr
+description: "The `attr()` CSS function returns the value of an HTML element's attribute, which can then be used as a typed value in a CSS declaration (types include length, number, angle, color, etc.). For example `width: attr(length);`. A default value may be provided if the attribute is missing or invalid."
```

Comment:
A few ideas here:

- Distinguish this from the content-only form of the feature
- Avoiding non-literal code samples (`width: attr(length);` — length is not, to the best of my knowledge, a defined attribute)
- Active voice on the fallback behavior

```suggestion
description: "The `attr()` CSS function, in the context of any property, returns the value of an attribute of an HTML element, with the option to return it as a specific type or unit. You can set a default value for missing or invalid attributes."
```